src/test/java/org/opensearch/knn/KNNSingleNodeTestCase.java

```diff
@@ -46,8 +46,17 @@
 import java.util.concurrent.ExecutionException;
 
 import static org.mockito.Mockito.when;
-import static org.opensearch.knn.common.KNNConstants.*;
```

Comment:
Pls avoid these wildcards

Reply:
Yes, the IDE automatically changed it...